CHANGELOG.md

```diff
@@ -1,3 +1,109 @@
+| ---------- | --------------------- |
+| bug fix |  **drag-drop:** pointer position calculation for SVG with viewBox ([#19863](https://github.com/angular/components/issues/19863)) ([78483bf](https://github.com/angular/components/commit/78483bf)) |
+| bug fix |  **platform:** rtl scroll axis incorrectly determined in Safari macOS ([#19830](https://github.com/angular/components/issues/19830)) ([0ddad07](https://github.com/angular/components/commit/0ddad07)), closes [#14609](https://github.com/angular/components/issues/14609) |
+| bug fix |  **schematics:** errors when attempting to read some files ([#19783](https://github.com/angular/components/issues/19783)) ([00c5a53](https://github.com/angular/components/commit/00c5a53)), closes [#19779](https://github.com/angular/components/issues/19779) |
+| bug fix |  **scrolling:** fixed-size-virtual-scroll wrong range and position when items length changes and current scroll is greater than new data length ([#19578](https://github.com/angular/components/issues/19578)) ([1d6ae58](https://github.com/angular/components/commit/1d6ae58)) |
+| feature |  **testing:** add base class for harnesses that may contain other harnesses ([#19944](https://github.com/angular/components/issues/19944)) ([6d2fb1f](https://github.com/angular/components/commit/6d2fb1f)) |
+| feature |  **tree:** add trackBy to NestedTreeControl ([#19602](https://github.com/angular/components/issues/19602)) ([e2c7e1a](https://github.com/angular/components/commit/e2c7e1a)) |
+
+### material
+
+|            |                       |
+| ---------- | --------------------- |
+| bug fix |  **a11y:** focus trap directive not capturing focus if auto capture input is set after init ([#19689](https://github.com/angular/components/issues/19689)) ([1acc608](https://github.com/angular/components/commit/1acc608)), closes [#19664](https://github.com/angular/components/issues/19664) |
+| bug fix |  **a11y:** list key manager not emitting change event if new item is added to active index ([#19666](https://github.com/angular/components/issues/19666)) ([3fbebe0](https://github.com/angular/components/commit/3fbebe0)), closes [#19661](https://github.com/angular/components/issues/19661) |
+| bug fix |  **aria-describer:** messages not being read out in IE and Edge ([#12304](https://github.com/angular/components/issues/12304)) ([0e22019](https://github.com/angular/components/commit/0e22019)), closes [#12298](https://github.com/angular/components/issues/12298) |
+| bug fix |  **autocomplete:** don't close when clicking inside custom origin ([#19784](https://github.com/angular/components/issues/19784)) ([1319c0a](https://github.com/angular/components/commit/1319c0a)), closes [#19771](https://github.com/angular/components/issues/19771) |
+| bug fix |  **benchmark:** fix license header and correct test descriptions ([#19596](https://github.com/angular/components/issues/19596)) ([0d266e2](https://github.com/angular/components/commit/0d266e2)) |
```

Comment:
I don't think these `benchmark` commits are supposed to show up here.

Reply:
Do we want `benchmark` to show up at all in the CHANGELOG?

Reply:
I don't think so, we probably should've marked these commits as `build`.

Reply:
SGTM - removed